db/migrate/20170118232430_create_join_table_authentication_configuration_script_base.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJoinTableAuthenticationConfigurationScriptBase < ActiveRecord::Migration[5.0]
+  def change
+    create_table :authentication_configuration_script_bases do |t|
+      t.bigint  :authentication_id
+      t.bigint  :configuration_script_base_id
+      t.index   :configuration_script_base_id, :name => 'configuration_script_base_id'
```

Comment:
Even, so prefer `index_configuration_script_base_id`  Naming the index the same as the column is guaranteed to cause confusion